api/docker-compose.yaml

```diff
@@ -75,7 +75,7 @@ services:
     volumes:
       - ..:/w:cached
       - gradle-cache:/.gradle
-    entrypoint: ['with-uid.sh', 'wait-for', 'db:3306', --]
+    entrypoint: ['with-uid.sh', 'wait-for-it', 'db:3306', --]
```

Comment:
`wait-for` is a specifically Alpine tool

Reply:
Running `docker-compose` with `wait-for` entrypoints fails with `Operation timed out`.